src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6563,4 +6563,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureVarianceSafetyForStaticInterfaceMembers" xml:space="preserve">
     <value>variance safety for static interface members</value>
   </data>
-</root>
+  <data name="WRN_AnalyzerReferencesFramework" xml:space="preserve">
+    <value>The assembly '{0}' containing type '{1}' references .NET Framework, which is not supported.</value>
+    <comment>{1} is the type that was loaded, {0} is the containing assembly.
+</comment>
```

Comment:
XML formatting got a bit screwed up here.

Reply:
F'ing resource files are the bane of my developer existence. (If only there were a better way 👍 )